automation/project/Dependencies.scala

```diff
@@ -43,7 +43,7 @@ object Dependencies {
     "com.typesafe.akka"   %%  "akka-http"           % akkaHttpV,
     "com.typesafe.akka"   %%  "akka-testkit"        % akkaV     % "test",
     "com.typesafe.akka"   %%  "akka-slf4j"          % akkaV,
-    "org.specs2"          %%  "specs2-core"   % "3.7"  % "test",
+    // "org.specs2"          %%  "specs2-core"   % "3.7"  % "test",
```

Comment:
`specs2` is not used in ui test suite. removing it in dependencies.

Reply:
any reason to keep it in here as a comment?

Reply:
it will be removed in final step before merge. no good to keep it.